src/main.c

```diff
@@ -592,7 +592,7 @@ int main(int argc, char* argv[]) {
     // Default search path list
     lib_search_paths = JV_ARRAY(jv_string("~/.jq"),
                                 jv_string("$ORIGIN/../lib/jq"),
-                                jv_string("$ORIGIN/lib"));
+                                jv_string("$ORIGIN/../lib"));
```

Comment:
So i understand: let's say the jq binary is at /usr/bin/jq then `$ORIGIN` will be replace with `/usr/bin` so search path will be `/usr/bin/../lib` abs path `/usr/lib`?